Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/components/content-manager/content-manager.component.ts

```diff
@@ -180,11 +180,9 @@ export class ContentManagerComponent extends PageBase implements OnInit, OnChang
       this.channel = channel;
 
       // Clears aggregation Filters, resets the aggregators
-      this.patchRequestState({
-        aggregationFilters: [],
-        aggregators: channel.aggregations,
-        channelId: channel.id,
-      });
+      this.facade.searchRequestState.aggregationFilters = [];
```

Comment:
By doing this are we not breaking the immutability of the state object?

Reply:
By removing the patchRequestState,  a search is not triggered at that point, it will, though, be triggered in the Typescript SDK side, in the ngOnChanges for the channel, where the request state is patched.